### PR TITLE
Update pubspec.yaml

### PR DESCRIPTION
**Pull request by CoreyRab**
Small tweak to the description on pub.dev.

### CODE REVIEW

**Review by aboedo — COMMENTED** [review]
we support macOS for flutter as well, right @vegaro?

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: purchases_flutter
-description: A Flutter plugin that makes it simple to build in-app purchases and subscriptions – receipt validation, status tracking, and more. Supports iOS and Android.
+description: A Flutter plugin that makes it simple to build in-app purchases (IAP) and subscriptions. Supports iOS, Android, and Stripe.
```
Comment (vegaro) [inline pubspec.yaml:2]: Would this lead to developers to think that we support the Web platform? We've had several requests for it https://github.com/RevenueCat/purchases-flutter/issues/162 . @RevenueCat/sdk what do you think?

**Comment by vegaro** [comment]
we do support MacOS @aboedo , we should probably add it yes, good call

File: pubspec.yaml (hunk 1 above)
Comment (aboedo) [reply pubspec.yaml:2]: It might also lead to some confusion since on our pub.dev, Web is marked as not supported, right?

Comment (CoreyRab) [reply pubspec.yaml:2]: Why is marked as not supported?

**Comment by aboedo** [comment]
I didn't realize that Flutter had watchOS support, they didn't list it in their platforms https://flutter.dev/docs/development/tools/sdk/release-notes/supported-platforms but they do have articles for it. 
I think we should list it, yes. 
As for iPadOS, happy to go either way - today it may be clear to everyone that everything supported by iOS is supported by iPadOS (since they're mostly the same), but if they start branching out more in the future then it may become less obvious over time and worth mentioning explicitly

**Review by vegaro — APPROVED** [review]

**Pull request merged**